x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -220,7 +226,7 @@ export const ActionForm = ({
         actionTypeId: actionTypeModel.id,
         group: defaultActionGroupId,
         params: {},
-        frequency: DEFAULT_FREQUENCY,
+        frequency: isSiem ? DEFAULT_SIEM_FREQUENCY : DEFAULT_FREQUENCY,
```

Comment:
We should add an optional prop to specify the default frequency, and specify `DEFAULT_SIEM_FREQUENCY` in the `security_solution` plugin.

Reply:
done